src/TraceEvent/RegisteredTraceEventParser.cs

```diff
@@ -979,6 +996,21 @@ private DynamicTraceEventData.PayloadFetchClassInfo ParseFields(int startField,
             ref int infoSize
         );
 
+        [DllImport("tdh.dll")]
+        internal static extern int TdhEnumerateManifestProviderEvents(
+            [In] ref Guid Guid,
+            [Out] byte[] pBuffer, // Actually PROVIDER_EVENT_INFO*
+            ref int pBufferSize
```

Comment:
Missing`[In, Out]` attributes?

Reply:
`ref` defaults to `[In, Out]`. I suppose I could make it explicit, but that just makes the binary larger.